lib/run-tutorial

```diff
@@ -1,10 +1,72 @@
 #!/bin/bash
+function usage() {
+  echo "Helpscreen for $(basename $0) (option parameter apply to tutorial bash scripts, too, see details)"
```

Comment:
Grammatically, should be either "option parameter applies", or "option parameters apply".
I would go with the following, though, unless "option parameter(s)" refers to something different:
```suggestion
  echo "Helpscreen for $(basename $0) (command line arguments apply to tutorial bash scripts, too, see details)"
```